### DISCUSSION TITLE
Dedicated Output - Sending Audio all the time

### DISCUSSION THREAD

**Discussion by rave2n** [DistroAV/DistroAV#821] [Q&A - Support]
I've been searching for a solution for this, and it appears it should have been solved a few versions back but it doesn't seem to reflect in my environment. 

Using the latest version of obs-ndi, with 4.5.1 NDI runtime. 

I use PTT on my gaming PC, but it receives audio regardless on the streaming PC. Certainly don't want a hot mic all the time!

From my readings, this is due to the filters being parsed before the actual output. 

Has anyone found a solution to this? Or should I just find a way to share a keypress?

Appreciate any knowledge I've missed!

**Comment by rave2n** [comment]
Found a workaround for my scenario. 

I use Voicemeter, so I just made a macro button to mute my input when not pressing my PTT hotkey, works great!

**Comment by Trouffman** [comment]
Should have been fixed for a while. Closing as part of a clean-up effort to keep only active/current versions/issues topic.
